Applications/Browser.md

```diff
@@ -141,14 +152,14 @@ Here, you can choose various parameters depending on the chosen Data Source. For
 
 <img src="_images/Filters.png" width="450">
 
-Fig 11: Data filters and parameters
+Fig 12: Data filters and parameters
 
 #### Visualize the search result
 
 Once you have found a product, you can visualize the results in two ways: either by directly selecting the viszualize button ( ![visualize](_images/icons/VisualizeButton.png) ) in the sidebar or by selecting the visualize button in the results panel on the map. You can open the results panel by clicking on one of the displayed tile footprints on the map.
 
 ![Search tab](_images/VisualizeTile.png)
-Fig 12: Product metadata and visualize button
+Fig 13: Product metadata and visualize button
```

Comment:
Can you please change this to <img src...> and then add a space between the image and the figure? The Figure title is currently not displayed nicely.